float64 theta
float64 theta_dot
float64 theta_ddot
